Makefile

```diff
@@ -130,24 +128,18 @@ docker-testfast: docker-build
 		cryptographic_estimators/base_problem.py \
 		cryptographic_estimators/estimation_renderer.py \
 		cryptographic_estimators/helper.py \
+		# cryptographic_estimators/SDEstimator/ \
 		" \
 		&& echo "All tests passed." \
 		|| echo "Some test have failed, please see previous lines."
-	@echo "Cleaning container..."
-	@make stop-container-and-remove container_name="container-for-test"
 
-add-copyright:
-	@python3 scripts/create_copyright.py
-
-docker-pytest:
-	@echo "Removing previous container...."
-	@make stop-container-and-remove container_name="pytest-estimators" \
+docker-pytest-doctests: CONTAINER_NAME := "pytest-container"
```

Comment:
very good idea! No need for sage in the doctests, which should significantly speed up the CI